hw03_ramav.py

```diff
@@ -0,0 +1,128 @@
+#################################################################################
+# Author:Valentin Rama
+# Username: ramav
+#
+# Assignment: HW03
+# Purpose: Creating a cat
+# Google Doc Link: https://docs.google.com/document/d/18V8i2I56Gj5T8IJGo4s0cEIvQE2YgXbE2uGVO0n7EVE/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+# TESTING PUSH FEATURE IN TA HOURS
+
+import turtle
+
```

Comment:
Pay attention to the gray squigglies, they indicate style guidelines. This one says there should be two blank lines before function definitions.